### PR TITLE
[UX2.0] Adds Policy Objects Profile Parcels - Part 2

### PR DESCRIPTION
**Pull request by seconroy**
### Description

Also adds the following Policy Objects profile parcel resource and data source:
- App Probe
- As Path
- VPN Group
- Security Data IP Prefix
- Security FQDN
- Security Geo Location
- Security IPS Signature
- Security Local Domain
- Security Port
- Security Scalable Group Tag
- Security Url List
- Preferred Color Group
- Security Identity
- Security Localapp
- Standard Community

### Types of Changes

* [x] New feature (non-breaking change which adds functionaality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Build/CI change
* [ ] Code quality improvement/refactoring/documentation (no functional changes)

### Checklist

* [x] My code follows the code style of this project
* [x] I have added tests to cover my changes
* [x] All new and existing tests pass locally

### CODE REVIEW

File: gen/definitions/profile_parcels/policy_object_security_ips_signature.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Policy Object Security IPs Signature
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_ips_signature.yaml:2]: Policy Object Security IPS Signature List (it's IPS for Intrusion Prevention System that's why it should be all capital).

File: gen/definitions/profile_parcels/policy_object_security_localapp.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Security Localapp
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_localapp.yaml:2]: Policy Object Security Local Application List

File: gen/definitions/profile_parcels/policy_object_security_localdomain.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Security Localdomain
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_localdomain.yaml:2]: Policy Object Security Local Domain List

File: gen/definitions/profile_parcels/policy_object_security_localdomain.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: var.policy_object_feature_template_id
+  - model_name: entries
+    mandatory: true
+    attributes:
+    - model_name: nameServer
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_localdomain.yaml:18]: tf_name: local_domain

File: gen/definitions/profile_parcels/policy_object_security_port.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Security Port
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_port.yaml:2]: Policy Object Security Port List

File: gen/definitions/profile_parcels/policy_object_security_scalable_group_tag.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+name: Policy Object Security Scalable Group Tag
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_scalable_group_tag.yaml:2]: Policy Object Security Scalable Group Tag List

File: gen/definitions/profile_parcels/policy_object_security_identity.yaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Security Identity
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_identity.yaml:2]: Policy Object Security Identity List

File: gen/definitions/profile_parcels/policy_object_security_urllist.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Security URLlist
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_security_urllist.yaml:2]: Policy Object Security URL List

File: gen/definitions/profile_parcels/policy_object_standard_community.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+---
+name: Policy Object Standard Community
```
Comment (tzarski0) [inline gen/definitions/profile_parcels/policy_object_standard_community.yaml:2]: Policy Object Standard Community List

**Review by tzarski0 — CHANGES_REQUESTED** [review]
I'm not sure if all comments make sense but I was trying to align it with: UX1.0 policy objects, UX2.0 policy objects we added in part 1 PR and UX2.0 names in UI.

File: gen/definitions/profile_parcels/policy_object_app_probe.yaml [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: var.policy_object_feature_template_id
+  - model_name: entries
+    mandatory: true
+    attributes:
+    - model_name: map
```
[the remark this answers is not in the document]
Comment (seconroy) [reply gen/definitions/profile_parcels/policy_object_app_probe.yaml]: With the current implementation this is to maintain the structure the API expects

**Review by tzarski0 — APPROVED** [review]

**Review by danischm — APPROVED** [review]

**Pull request merged**